chrome/content/zotero/preferences/preferences_sync.js

```diff
@@ -213,11 +213,22 @@ Zotero_Preferences.Sync = {
 
 		this.displayFields();
 		Zotero.Prefs.clear('sync.librariesToSync');
+		var userLibraryID = Zotero.Libraries.userLibraryID;
```

Comment:
I don't believe this is actually needed anymore with the pruning done in `initLibraryAttachmentBasePaths`.

Reply:
Testing showed this cleanup is not necessary.